parsers/CL.py

```diff
@@ -102,53 +102,16 @@ def production_processor_historical(raw_data):
         combined[elem['datetime']].update(elem)
 
     ordered_data = sorted(combined.values(), key=itemgetter("datetime"))
-
-    # For consistency with live API, hydro and geothermal must be squeezed into unknown
```

Comment:
Dont' we still need to do this if we enable the live parser again? In that case we could move `ENABLE_LIVE_PARSER` to the outer scope and check it here.

Reply:
I was thinking that if we enable the live parser again, we would have a system to keep both datasets. I.e. we don't want to reenable the live parser if that means we have to lump hydro and geothermal into unknown.

But, I can see how it would be confusing in the meantime. I've added the check